packages/flutter_driver/lib/src/extension/extension.dart

```diff
@@ -372,12 +427,30 @@ class FlutterDriverExtension {
   }
 
   Finder _createFinder(SerializableFinder finder) {
-    final FinderConstructor constructor = _finders[finder.finderType];
-
-    if (constructor == null)
-      throw 'Unsupported finder type: ${finder.finderType}';
-
-    return constructor(finder);
+    switch (finder.finderType) {
```

Comment:
I think we should move this logic to our default `DeserializeFinderFactory`, and make it `@mustCallSuper`.

Reply:
The main purpose of using DeserializeFinderFactory here is to decouple from "dart:ui", so I think it should not be added